app/genres/fiori-service.cds

```diff
@@ -0,0 +1,23 @@
+/*
+  Annotations for the Browse GenreHierarchy App
+*/
+
+using AdminService from '../../srv/admin-service';
+
+
+annotate AdminService.GenreHierarchy with @Aggregation.RecursiveHierarchy#AdminHierarchy: {
+    $Type: 'Aggregation.RecursiveHierarchyType',
+    NodeProperty: node_id, // identifies a node
+    ParentNavigationProperty: parent // navigates to a node's parent
+  };
+
+  annotate AdminService.GenreHierarchy with @Hierarchy.RecursiveHierarchy#AdminHierarchy: {
+  $Type: 'Hierarchy.RecursiveHierarchyType',
+  // ExternalKey           : null,
+  LimitedDescendantCount: LimitedDescendantCount,
+  DistanceFromRoot: DistanceFromRoot,
+  DrillState: DrillState,
+  Matched: Matched,
+  MatchedDescendantCount: MatchedDescendantCount,
+  LimitedRank: LimitedRank
```

Comment:
we need to check if we need this annotation if we don't rename anything

Reply:
I agree, it looks the same in the example I got from Gwendal (no renaming), but I can't find confirmation in the documentation -> ask UI colleagues